model/src/main/proto/interaction_object.proto

```diff
@@ -0,0 +1,69 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.app.model";
+option java_multiple_files = true;
+
+message InteractionObject {
+  oneof object {
```

Comment:
Maybe: ``oneof object_type``. Thoughts?

Reply:
I have added an object type, so changed this field to value - what do you think?

Reply:
This wasn't quite what I meant--followed up below. Let me know if you want to discuss further.